src/appsettings.template.json

```diff
@@ -6,7 +6,10 @@
         "BypassStringTypeCoercion": "${BYPASS_STRING_COERCION}"
     },
     "ConnectionStrings": {
-        "DatabaseConnection": "host=${POSTGRES_HOST};port=${POSTGRES_PORT};username=${POSTGRES_USER};password=${POSTGRES_PASSWORD};database=edfi_datamanagementservice;"
+        "DatabaseConnection": "${DATABASE_CONNECTION_STRING}"
+    },
+    "DatabaseOptions": {
+        "IsolationLevel": "${DATABASE_ISOLATION_LEVEL}"
```

Comment:
It could be done either way but I isolated it here because the corresponding c# class in the backend project only needed this one property. It just felt cleaner in my brain to have `"DatabaseOptions" {...}` map to `DatabaseOptions.cs`